G1ANT.Addon.UI/G1ANT.Addon.UI/Panels/UIControlsPanel.cs

```diff
@@ -121,5 +134,130 @@ private void refreshButton_Click(object sender, EventArgs e)
+                        if (element != null)
+                        {
+                            var window = GetTopLevelWindow(automationElement);
+                            if (window != null)
+                            {
+                                var iHandle = new IntPtr(window.Current.NativeWindowHandle);
+                                if (iHandle != IntPtr.Zero)
+                                {
+                                    RobotWin32.BringWindowToFront(iHandle);
+                                }
+                            }
+                            var rect = element.GetRectangle();
+                            if (rect != null) { initializeRectangleForm(rect); }
+                        }
+                    }
+                }
+            }
+            catch (Exception ex)
+            {
+                RobotMessageBox.Show(ex.Message,"Error");
+            }
+        }
+        
+        private Timer blinkTimer;
+        private void ClosingTimer_Tick(object sender, EventArgs e)
+        {
+            closingTimer.Enabled = false;
+            blinkTimer.Enabled = false;
+            rectangleForm.Close();
+        }
+
+        private void initializeRectangleForm(System.Windows.Rect rect)
+        {
+            rectangleForm = new Form();
+            Panel transparentPanel = new Panel();
+            transparentPanel.BackColor = Color.Pink;
+            transparentPanel.Location = new Point(3, 3);
+            transparentPanel.Padding = new System.Windows.Forms.Padding(30);
+            transparentPanel.Parent = rectangleForm;
+            rectangleForm.Controls.Add(transparentPanel);
+            rectangleForm.TransparencyKey = Color.Pink;
+            rectangleForm.BackColor = Color.Red;
+            rectangleForm.ForeColor = Color.Red;
+            rectangleForm.TopMost = true;
+            rectangleForm.FormBorderStyle = FormBorderStyle.None;
+            rectangleForm.ControlBox = false;
+            rectangleForm.Text = string.Empty;
+            rectangleForm.StartPosition = FormStartPosition.Manual;
+            rectangleForm.MinimumSize = new Size(10, 10);
+            rectangleForm.Location = new Point((int)rect.Left, (int)rect.Top);
+            rectangleForm.Size = new Size((int)(rect.Right - rect.Left), (int)(rect.Bottom - rect.Top));
+
+            transparentPanel.Size = new Size(rectangleForm.Size.Width - 6, rectangleForm.Size.Height - 6);
+            rectangleForm.Shown += RectangleForm_Shown;
+            rectangleForm.Show();
+        }
+
+        private Timer closingTimer;
+        private void RectangleForm_Shown(object sender, EventArgs e)
+        {
+            blinkTimer = new Timer();
+            blinkTimer.Interval = 300;
+            blinkTimer.Tick -= BlinkTimer_Tick;
+            blinkTimer.Tick += BlinkTimer_Tick;
+            blinkTimer.Enabled = true;
+
+            closingTimer = new Timer();
```

Comment:
Do we really need second timer which will close blinking form? Can we do this in BlinkTimer_Tick after 5 "blinks"?